### DISCUSSION TITLE
Crossing the Chasm in Comprehension of OBS

### DISCUSSION THREAD

**Discussion by KijanaG** [obsproject/obs-studio#6449] [General]
Hey there all, I have the goal to eventually create an OBS plugin but am stuck (intimidated) trying to understand the library & what the code is actually doing. 

I have ~3 years coding experience – mainly in JS & Python, a class or two in C(++). 

Any suggestions on how to ramp up on learning the different parts of this library and what's happening behind the scenes?

Thanks!

KJ

**Comment by Fenrirthviti** [comment]
https://obsproject.com/wiki/Getting-Started-with-OBS-Studio-Development

Most casual development discussion takes place in our [Discord](https://obsproject.com/discord).

**Reply by KijanaG** [reply]
Hey, thanks so much!
